packages/datagateway-dataview/src/breadcrumbs.component.tsx

```diff
@@ -248,12 +251,9 @@ class PageBreadcrumbs extends React.Component<
         base: {
           entityName: baseEntityName,
 
-          // TODO: This display name requires internationalisation,
-          //       as currently it adds an 's' to the word disregarding
-          //       the current language the application is being served in.
-          displayName:
-            `${baseEntityName}`.charAt(0).toUpperCase() +
-            `${baseEntityName}s`.slice(1),
+          displayName: this.props.t(`breadcrumbs.${baseEntityName}`, {
```

Comment:
There's something going wrong with the breadcrumb label for facility cycles
![image](https://user-images.githubusercontent.com/18294786/88202694-28f05100-cc41-11ea-9a71-0d93d811c352.png)